cmd/cli/rekor/rekoruuid_validator.go

```diff
@@ -41,10 +41,10 @@ func Validate(vendorFile config.VendorFile, downloadedFile string) (err error) {
 			continue
 		}
 
-		decoded, err := base64.StdEncoding.DecodeString(string(entry.Attestation.Data))
+		decoded, err := base64.StdEncoding.DecodeString(entry.Attestation.Data.String())
```

Comment:
This one feels wrong. The `.String()` method runs:

```
return base64.StdEncoding.EncodeToString([]byte(b))
```

so I think we may be able to skip the base64 round trip and consider the Data already decoded. But that assumes Base64 doesn't change it's implementation.